core/error_api.php

```diff
@@ -247,11 +247,16 @@ function error_handler( $p_type, $p_error, $p_file, $p_line, array $p_context )
 
 	$t_error_description = nl2br( $t_error_description );
 
+	# Make sure the file path is not disclosed via exception details
+	$t_error_description = str_replace( config_get_global( 'absolute_path' ), '.../', $t_error_description );
```

Comment:
we have
```
define ('DIRECTORY_SEPARATOR', "/");
$g_absolute_path = dirname( __FILE__ ) . DIRECTORY_SEPARATOR;
```
Can't check myself at the moment, did you try if it works on Windows?

Should the path be removed if `ON == config_get_global( 'show_detailed_errors' )`?

Reply:
I initially had the code to only show the path when `show_detailed_errors` is ON.  But I then removed this code since I think it is not worth having the extra checks.  The real value is the relative path.

I don't have a Windows environment setup at the moment.

Reply:
I can test on Windows when I'm in the office on Monday.